sox.features.js

```diff
@@ -2620,6 +2620,26 @@
       const answers = document.querySelector("#answers-header h2").innerText.split(' ')[0];
       const toInsert = '<div class="grid--cell ws-nowrap mb8 ml16"><span class="fc-light mr4">Answers</span> ' + answers + '</div>';
       document.querySelector(".d-flex.fw-wrap").insertAdjacentHTML("beforeEnd", toInsert);
-    }
+    },
+
+    customizeToolsPageLists: function(settings) {
+        const trimLists = function(table) {
+            if (settings.filterInvalid) {
+                table.querySelectorAll("td.tagged-ignored").forEach(cell => cell.parentNode.remove());
+            }
+            if (settings.filterTypeQuestions) {
+                table.querySelectorAll("a.question-hyperlink").forEach(cell => cell.parentNode.parentNode.remove());
+            }
+            if (settings.filterTypeAnswers) {
+                table.querySelectorAll("a.answer-hyperlink").forEach(cell => cell.parentNode.parentNode.remove());
+            }
+            const listCount = settings.listCount || 3;
+            const rows = table.querySelectorAll("tr");
+            rows.forEach((row, i) => row.classList.toggle("collapsing", i >= listCount));
+        };
+        // tables are populated by XHR after page loads
+        sox.helpers.observe(Array.from(document.querySelectorAll("div.island")), "table.summary-table", trimLists);
+        // just in case we come in after a table has been loaded already
+        document.querySelectorAll("table.summary-table").forEach(table => trimLists(table));
   };
```

Comment:
There's a missing `}` just above this line, to close the new `customizeToolsPageLists` function